_events/2020-01-21-MachineLearning.md

```diff
@@ -0,0 +1,109 @@
+---
+site: freiburg
+tags:
+- training
+title: Machine learning workshop in Galaxy
+starts: 2020-03-30
+ends: 2020-04-01
+organiser:
+  name: Freiburg Galaxy Team
+  email: contact@usegalaxy.eu
+location:
+  name: PC Pool 3
+  street: Werthmannstraße 4 
+  postal: 79098
+  city: Freiburg im Breisgau
+  region: Baden-Württemberg
+  country: Germany
+  geo:
+    lat: 47.993469
+    lon: 7.845275
+supporters:
+- unifreiburg
+- denbi
+- elixir
+# special hiding of footer since we want to do it manually.
+hidefooter: true
+---
+
+We are offering workshop on introduction to machine learning using Galaxy. 35 participants will be selected from all applicants. To apply for attendance, please sign in [here](https://docs.google.com/forms/d/e/1FAIpQLSdu1SJbSOCGy20iqShiyK5DUIrZU6hnPGpYv2DgHnUxf13HSQ/viewform?usp=pp_url).
+
+## Venue
+
+{% include map.html location=page.location showmap=true zoomlevel=15 hidepopup=true %}
+
+
+# Important notes
+
+1. If you are registered but **cannot attend** our workshop, please [contact us
+   by email](mailto:contact@usegalaxy.eu) immediately. We have a long waiting
+   list and can give your place to others, even on short notice.
+2. Please register to our [European Galaxy server](https://usegalaxy.eu) to perform the analysis.
+3. You can bring your *own notebook* or desktop computers will be available. Eduroam is available, ask your institute 
+   for how to login.
+4. The workshop is free of charge. No stipends for travel or accommodation are available.
+
+## Program:
+
+Every day the workshop will run from 9:00-17:00 (give or take, depending on questions at the end). If the times will change, it will be announced during the workshop.
+
+Preliminary schedule. Topics may be adjusted according to the participants demand
+
+Day     | Topics
+------- | --------
+Mon     | **Starts at 10:00 am**, Introduction to Galaxy, machine learning basics and classification techniques.
+Tue     | Regression models, unsupervised learning (clustering and dimension reduction) and feature selection methods.
+Wed     | Hyperparameter optimization techniques and bring your own data as a usecase, questions and discussions. **Ends at 2:00 pm**
```

Comment:
@khanteymoori the bring-your-own-data is dangerous. You might raise here expectations which we can not fulfill.

Reply:
Deleted.